tools/template/create.sh

```diff
@@ -29,7 +29,7 @@ fi
 
 
 #dont put pgr_ on MY_FUNCTION_NAME that is done automatically
-MY_FUNCTION_NAME="funnyDijkstra"
+MY_FUNCTION_NAME="funny_dijkstra"
```

Comment:
Next time you want get the original file do a:
git checkout -- <name of file>
No harm done with this change